lib/widgets/app_header.dart

```diff
@@ -1,26 +1,50 @@
+import 'package:firebase_auth/firebase_auth.dart';
 import 'package:flutter/material.dart';
+import 'package:get/route_manager.dart';
+import 'package:regatta_buddy/pages/login_page.dart';
+import 'package:regatta_buddy/pages/profile_page.dart';
 
-class AppHeader extends StatelessWidget implements PreferredSizeWidget {
+class AppHeader extends StatefulWidget implements PreferredSizeWidget {
```

Comment:
trochę nie widzę czemu to jest stateful? co jest tutaj do pamiętania?

Reply:
no chyba może być stateless

Reply:
on jest stateful bo w zaleznosci od stanu uwierzytelnienia wyswietlana jest odpowiednia ikona. patrz `getAuthButton(bool showAuthButton)`